### ISSUE TITLE
Need to connect edx analytics dashboard (insights) to analytics pipeline

### ISSUE THREAD

**Issue by testaccountsnt** [openedx-unsupported/edx-analytics-pipeline#716]
I have installed edx analytics pipeline and it is running successfully.

    edxops/analytics_pipeline:latest
    edxops/analytics_pipeline_spark_worker:latest
    edxops/analytics_pipeline_hadoop_nodemanager:latest
    edxops/analytics_pipeline_hadoop_resourcemanager:latest
    edxops/analytics_pipeline_hadoop_datanode:latest
    edxops/analytics_pipeline_spark_master:latest
    edxops/analytics_pipeline_hadoop_namenode:latest

But when I open insights and login to it, it shows following message after login.
unauthorized_client An unauthorized client tried to access your resources.

I have created client id in lms admin and added them as Trusted client.
Now I need to add key in insights.yml. Please help me to find insights.yml in edx analytics docker. How can I loginto the instance and where can I find this file.

Apart from it if anything else is needed to fic the issue. Please let me know.

Thanks

**Comment by somanshreddy** [comment]
Hi, 
     This is not an answer to your question. I actually have an issue that I would like to ask you. How did you install edx analytics pipeline? Did you use Docker? 

Thanks,
Somansh

**Comment by testaccountsnt** [comment]
Yes that is correct, I have installed it using Docker (devstack installation) because I need ecommerce and Insights (Analytics). I installed before using bitnami installer (https://bitnami.com/stack/edx/installer) and started using lms and studio, all was working well until I get requirement of ecommerce (Paid courses with cart functionality and payment module) and Insights (For data visibility). I tried with core installation but it was not a success because of its lengthy procedure. Using docker I managed to get lms, studio and ecommerce worked but Insights is still not working. Pipeline is running successfully but it does not proceed further after login.
